### DISCUSSION TITLE
leveling up the terrain

### DISCUSSION THREAD

**Discussion by SgtOtaku** [scfmod/FS22_TerraFarm#67] [Q&A] [answered]
I'm loving this mod so far, but one odd bit of behavior that i've experienced is that when using any tool, even modded equipment, the leveling function will only bring terrain DOWN to level it, but will not RAISE a dip to be level with the rest. is there some sort of button I'm supposed to press to make it work like that, or do i need to keep raising the ground, and THEN leveling it to make it work?

**Comment by scfmod** [comment, marked as answer]
The leveling function uses the terrain height at vehicle root position (usually around the center of the vehicle). There may be more options for it later on :-)
